js/bybit.js

```diff
@@ -3098,9 +3098,12 @@ module.exports = class bybit extends Exchange {
         if (price !== undefined) {
             request['p_r_price'] = this.priceToPrecision (symbol, price);
         }
-        const triggerPrice = this.safeValueN (params, [ 'stopPrice', 'triggerPrice', 'stop_px' ]);
-        if (triggerPrice !== undefined) {
-            request['p_r_trigger_price'] = this.priceToPrecision (symbol, triggerPrice);
+        if (isConditionalOrder) {
+            const triggerPrice = this.safeValueN (params, [ 'stopPrice', 'triggerPrice', 'stop_px' ]);
+            if (triggerPrice !== undefined) {
+                request['p_r_trigger_price'] = this.priceToPrecision (symbol, triggerPrice);
+                params = this.omit (params, [ 'stopPrice', 'triggerPrice', 'stop_px' ]);
```

Comment:
In this case the exchange-specific parameter name is `p_r_trigger_price`, instead of `stop_px`, so there should be no `stop_px` here.